waku/v2/node/wakunode2.nim

```diff
@@ -128,7 +128,8 @@ template tcpEndPoint(address, port): auto =
 proc new*(T: type WakuNode, nodeKey: crypto.PrivateKey,
     bindIp: ValidIpAddress, bindPort: Port,
     extIp = none[ValidIpAddress](), extPort = none[Port](),
-    peerStorage: PeerStorage = nil): T 
+    peerStorage: PeerStorage = nil,
+    maxConnections = MaxConnections): T 
```

Comment:
Could not find the `MaxConnections` declaration? is it added in this PR?

Reply:
Wondering where the `maxConnection` gets checked?

Reply:
Hah. Good point.

It's an underlying `switch` configuration variable (we try and keep these hidden, unless we actually need to configure one). This one is specifically defined and [checked in the `libp2p` `connmanager`](https://github.com/status-im/nim-libp2p/blob/75bfc1b5f7679afc104bd1ceee1a0dc3bab7a316/libp2p/connmanager.nim#L97). `MaxConnections` is also [defined there](https://github.com/status-im/nim-libp2p/blob/75bfc1b5f7679afc104bd1ceee1a0dc3bab7a316/libp2p/connmanager.nim#L27). Since we import that module (via `builders`) we can use it here. I've tried to make it a little bit clearer now, by adding the imported module as namespacing.